test/integration/backward_compatible/parallel/map/NearCachedMapTest.js

```diff
@@ -100,11 +101,19 @@ describe('NearCachedMapTest', function () {
                 if (!invalidateOnChange) {
                     this.skip();
                 }
+
                 await map1.get('key1');
                 await map2.remove('key1');
-                const val = await TestUtil.promiseLater(5000, map1.get.bind(map1, 'key1'));
-                expectStats(map1, 0, 2, 1);
-                expect(val).to.be.null;
+
+                let previousHitCount;
+
+                await assertTrueEventually(async () => {
+                    previousHitCount = getNearCacheStats(map1).hitCount;
+                    const val = await map1.get('key1');
+                    // Client should eventually use near cache
```

Comment:
I think this comment is not correct. Client should eventually ask the cluster for this key, after seeing that its local copy is invalidated

Reply:
Yes and then it should keep asking from near cache. What about `// Client should eventually fetch invalidated entry from cluster and then start using its near cache `

Reply:
This should "eventually use near cache" refers to that

Reply:
`// Client should eventually fetch invalidated entry from cluster `
I think this part is enough. In this test, I only care about the invalidation

Reply:
🤔  Then we should assert that missCount increases in the assertTrueEventually? I am trying to align comment and code

like this:

```js
await assertTrueEventually(async () => {
                    const previousHitCount = getNearCacheStats(map1).missCount;
                    const val = await map1.get('key1');
                    // Client should eventually fetch invalidated entry from cluster due to invalidation
                    expect(getNearCacheStats(map1).missCount).to.be.greaterThan(previousHitCount);
                    expect(val).to.be.null;
                }, 1000);
```

Reply:
Yes, the code seems correct to me. Or simply
```js
await assertTrueEventually(async () => {
                    const val = await map1.get('key1');
                    // Client should eventually fetch invalidated entry from cluster due to invalidation
                    expect(getNearCacheStats(map1).missCount).to.be.greaterThan(1);
                    expect(val).to.be.null;
                }, 1000);
```

Reply:
ok applying. I was behaving safe, but I think there is no way the missCount to be greater than 1 before we start executing assertTrueEventually.